echidna/scripts/zero_nu_limit.py

```diff
@@ -85,18 +84,49 @@ def __call__(self, parser, namespace, values, option_string=None):
     shrunk = Te130_0n2b.sum()
     scaling = shrunk/unshrunk
 
-    Te130_0n2b = store.load(args.signal)
+    # Set decay converter
+    atm_weight_iso = 129.9062244
+    atm_weight_nat = 127.603
+    abundance = 0.3408
+
+    phase_space = 3.69e-14  # Kotila & Iachello 2012
+    matrix_element = 4.03
+
+    converter = decay.DBIsotope(
+        "Te130", atm_weight_iso, atm_weight_nat, abundance, phase_space,
+        matrix_element, roi_efficiency=scaling)
+
+    Te130_0n2b = store.load(args.signal)  # Reload spectrum
 
     # Create background spectra
     Te130_2n2b = store.load(args.two_nu)
-    B8_Solar = store.load(args.b8_solar)
+    Te130_2n2b.scale(1.e6)
+    unshrunk = Te130_2n2b.sum()
+    Te130_2n2b = store.load(args.two_nu)
+    Te130_2n2b.scale(1.e6)
+    shrink_dict = {"energy_mc_low": roi[0], "energy_mc_high": roi[1]}
+    Te130_2n2b.shrink(**shrink_dict)
+    shrunk = Te130_2n2b.sum()
+    scaling = shrunk / unshrunk
```

Comment:
As we agreed I think this should be kept so that the `DBIsotope` converter is initialised correctly with with `roi_efficiency`. Although the script doesn't currently use any of this functionality, I think it is important to correctly initialise it in-case anyone wishes to extend the script and then requires this functionality.